docs/api.rst

```diff
@@ -816,6 +816,9 @@ of :class:`enum.Enum`.
     .. attribute:: group
 
         A private group text channel.
+    .. attribute:: category
+
+       A category channel.
```

Comment:
This indentation is off:
```suggestion
        A category channel.
```

Reply:
Sorry, not sure how I didn't see that 🤦‍♂